docs/LowerBound_Dist_Profile_Derivation.ipynb

```diff
@@ -0,0 +1,2319 @@
+{
```

Comment:
<p>Equation (4) seems redundant here. Also, I think it would be better to transform equation (1) by first substitution <code>\alpha_t</code> and then to <code>f</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I will change it,